test/compiler.test.js

```diff
@@ -2234,6 +2286,15 @@ describe('compiler', function() {
           env: 'applied'
         }
       });
+    }
+
+    it('loads component configs from multiple files', function() {
+      testComponentConfigsMerge();
+    });
+
+    it('Suports `componentRootDir` Option And ' +
+      'loads component configs from multiple files', function() {
+      testComponentConfigsMerge(true);
```

Comment:
Similarly here. No need to test support for environment overrides, write a simple unit-test that verifies that `component-config.json` is loaded from the configured location.